lib/ecto/migration.ex

```diff
@@ -768,9 +772,15 @@ defmodule Ecto.Migration do
 
       execute "CREATE EXTENSION postgres_fdw", "DROP EXTENSION postgres_fdw"
 
+      alias Ecto.Adapters.SQL
```

Comment:
We need a bit more structure around this code example. Maybe add some leading paragraph about how it works and define the whole migration file, so the `defp` makes sense.

Reply:
I will add example here, but I believe that it would be nice to place such paragraph inside `Flushing`  section which would show difference between those two functions.